nixos/modules/config/nix.nix

```
@@ -95,14 +97,19 @@ let
 
       mkKeyValuePairs = attrs: concatStringsSep "\n" (mapAttrsToList mkKeyValue attrs);
 
+      isExtra = key: hasPrefix "extra-" key;
+
     in
     pkgs.writeTextFile {
       name = "nix.conf";
+      # workaround for https://github.com/NixOS/nix/issues/9487: extra-* settings
```

```suggestion
      # workaround for https://github.com/NixOS/nix/issues/9487 extra-* settings
```
GitHub linkifies that wrong 😅

---

done